src/models/Settings.js

```diff
@@ -25,6 +32,10 @@ function (Okta, Errors, BrowserFeatures) {
       ConfigError = Errors.ConfigError,
       UnsupportedBrowserError = Errors.UnsupportedBrowserError;
 
+  var assetBaseUrlTpl = Okta.tpl(
+    'https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/{{version}}'
```

Comment:
General idea:
- By default, will be serving off the CDN
- In dev, I made a change to the Gruntfile to point assetBaseUrl to our local files
- Anyone that's running off latest master (or a version whose published assets have not hit production) will need to override the assetBaseUrl. This is what we'll need to do as well in core code since we're running off early builds

Do you foresee any problems with this?

*Note:* I do think it will be good to think about a faster deploy-to-cdn process for published artifacts in the near future (along with this, we still have the problem of syncing dev.okta.com docs)